cmd/memtierd/README.md

```diff
@@ -276,17 +276,85 @@ ssh debian@172.17.0.2 "sudo mv meme /usr/local/bin"
 
 ## Policies
 
-Memtierd implements two policies: age and heat. These policies have
-different ways of interpreting memory tracker counters as page
-activity.
-
-Policies measure and manage the memory of processes that are defined
-in the configuration. Processes are searched by pidwatchers. The
-cgroups pidwatcher looks for processes under listed a list of cgroups
-directories, while the pidlist pidwatcher passes a static list of pids
-to policies. See
+Memtierd implements three policies: age, heat, and ratio. Age and heat
+policies move or swap out memory based on last access times (age) or
+memory activity class (heat class). The ratio policy moves or swaps
+out a fixed ratio of least recently used memory.
+
+## Watchers
+
+Policies track and manage the memory of processes of
+interest. Processes are found and filtered by pidwatchers. The cgroups
+pidwatcher looks for processes under listed cgroups directories. The
+proc pidwatcher finds all running processes in the system. The pidlist
+pidwatcher finds only the static list of pids. The filter pidwatcher
+filters interesting processes out of all processes found by other
+pidwatchers.
+
+For examples of configuring pidwatchers as part of policy
+configuration, see:
+
 [memtierd-age-idlepage-trackonly.yaml](../../sample-configs/memtierd-age-idlepage-trackonly.yaml).
 
+### Pidwatcher: cgroups
+
+Finds and watches pids under listed cgroups directories.
+
+Configuration options:
+- `intervalms` (int): poll interval (ms) for changes in pids in listed
+  cgroups.
+- `cgroups` (list of strings): list of absolute paths to cgroup
```

Comment:
use "cgroups directories" instead of  "cgroup directories", to keep it aligned with L301.

Reply:
Thanks, fixed.